docs/reference/docfx-environment-variables-reference.md

```diff
@@ -14,6 +14,11 @@ If set true. Keep following debug info in output HTML.
 
 Used to override git branch name.
 
+## `DOCFX_SOURCE_REPOSITORY`
+
+Used to override git organization and repository names.
+It must be defined in the `{organization}/{repository_name}` format (e.g.`dotnet/docfx`).
```

Comment:
`{owner}/{repo}` is a pattern for GitHub but doesn't seem to fit every other repo management systems. 

As this is a configuration exposed to public, I would prefer it to be `DOCFX_SOURCE_REPOSITORY_URL` but we only support GitHub URLs (`https://github.com/{owner}/{repo}`) as a starting point. With that you can pattern match the URL in `ResolveDocfxSourceRepoUrl` method and return the original URL if  it is not github.

Reply:
I've changed environment variable name  to `DOCFX_SOURCE_REPOSITORY_URL` by commit (fa41fed).

URL is replaced when
- `DOCFX_SOURCE_REPOSITORY_URL` environment variable is defined.
- URL host name is matched to `DOCFX_SOURCE_REPOSITORY_URL`.
- Hostname matches one of the following URLs (that use `{org_name}/{repo_name}` patterns)
  - `github.com`
  - `bitbucket.org`
  - `dev.azure.com`